src/home_robot/home_robot/utils/point_cloud.py

```diff
@@ -53,30 +46,114 @@ def show_pcd(
     orig: np.ndarray = None,
     R: np.ndarray = None,
     save: str = None,
-    grasps: list = [],
+    grasps: list = None,
```

Comment:
Btw using a list as a default parameter is discouraged, because if the function modifies the list (e.g. appends something to it), then all calls *to* the function will now default to having that modified list, because of how python initializes default parameters.

Reply:
Good catch, thanks!